lib/utils/isMathFunction.js

```diff
@@ -0,0 +1,17 @@
+/* @flow */
+"use strict";
+
+const MATH_FUNCTIONS = ["calc"];
+
+/**
+ * Check whether a node is math function
+ *
+ * @param {Node} postcss-value-parser node
+ * @return {boolean} If `true`, the node is math function
+ */
+module.exports = function isMathFunction(node /*: Object */) /*: boolean*/ {
+  return (
+    node.type === "function" &&
+    MATH_FUNCTIONS.indexOf(node.value.toLowerCase()) > -1
```

Comment:
We can use this `MATH_FUNCTIONS.includes(node.value.toLowerCase())`, `indexOf` is very old, we can use more readable code :smile: 

And move `MATH_FUNCTIONS` to https://github.com/stylelint/stylelint/tree/master/lib/reference for better maintenance

Reply:
done